numpy/lib/function_base.py

```diff
@@ -2466,7 +2466,7 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
     Estimate a covariance matrix, given data and weights.
 
     Covariance indicates the level to which two variables vary together.
-    If we examine N-dimensional samples, :math:`X = [x_1, x_2, ... x_N]^T`,
+    If we examine N-dimensional samples, :math:`X = [x_1, x_2, \\dots, x_N]^T`,
```

Comment:
I'd err on the side of making this an `r""" ⋯ """` docstring rather than adding backslashes